clang/test/CodeGenHLSL/builtins/dot2add.hlsl

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cc1 -finclude-default-header -fnative-half-type -triple \
-// RUN:   dxil-pc-shadermodel6.3-compute %s -emit-llvm -o - | \
+// RUN:   dxil-pc-shadermodel6.4-compute %s -emit-llvm -o - | \

```

Comment:
Should we add an error test for this DXILOp so if we see it for SM 6.3 we  can confirm we let the user know?